lib/elixlsx/xml_templates.ex

```diff
@@ -300,11 +296,29 @@ defmodule Elixlsx.XMLTemplates do
     """
   end
 
+  defp make_data_validation({start_cell, end_cell, date: [date_validation]}) do
+    {condition, date} = date_validation
+
+    operator =
+      case condition do
+        :greater_than_or_equal -> "greaterThanOrEqual"
```

Comment:
Let's keep an eye on this sort of thing. If we need more of it, perhaps we should use a library like [inflex](https://github.com/nurugger07/inflex)